common/src/main/java/com/wynntils/models/containers/ContainerModel.java

```diff
@@ -64,11 +108,42 @@ public boolean isLootOrRewardChest(String title) {
         return isLootChest(title) || isRewardChest(title);
     }
 
-    public boolean isSeaskipper(Component component) {
-        return StyledText.fromComponent(component).equals(SEASKIPPER_TITLE);
+    public WynncraftContainer getCurrentContainer() {
+        return currentContainer;
     }
 
     public Matcher lootChestMatcher(Screen screen) {
         return StyledText.fromComponent(screen.getTitle()).getNormalized().getMatcher(LOOT_CHEST_PATTERN);
     }
+
+    public boolean isSeaskipper(Component component) {
+        return StyledText.fromComponent(component).equals(SEASKIPPER_TITLE);
+    }
+
+    private void registerWynncraftContainers() {
+        // Order does not matter here so just keep it alphabetical
+        registerWynncraftContainer(new AbilityTreeContainer());
+        registerWynncraftContainer(new AccountBankContainer());
+        registerWynncraftContainer(new BlockBankContainer());
+        registerWynncraftContainer(new BookshelfContainer());
+        registerWynncraftContainer(new CharacterBankContainer());
+        registerWynncraftContainer(new ContentBookContainer());
+        registerWynncraftContainer(new GuildBankContainer());
+        registerWynncraftContainer(new GuildMemberListContainer());
+        registerWynncraftContainer(new GuildTerritoriesContainer());
+        registerWynncraftContainer(new HousingJukeboxContainer());
+        registerWynncraftContainer(new HousingListContainer());
+        registerWynncraftContainer(new JukeboxContainer());
+        registerWynncraftContainer(new LobbyContainer());
+        registerWynncraftContainer(new MiscBucketContainer());
+        registerWynncraftContainer(new PetMenuContainer());
+        registerWynncraftContainer(new ScrapMenuContainer());
+        registerWynncraftContainer(new TradeMarketFiltersContainer());
+        registerWynncraftContainer(new TradeMarketPrimaryContainer());
+        registerWynncraftContainer(new TradeMarketSecondaryContainer());
```

Comment:
Maybe we could keep it alphabetical by type, kind of like features. But we can keep it as is, really doesn't matter.